.azure-pipelines/README.md

```diff
@@ -1,3 +1,7 @@
+
+[comment]: # GNU General Public License v3.0+ (see LICENSES/GPL-3.0-or-later.txt or https://www.gnu.org/licenses/gpl-3.0.txt)
+[comment]: # SPDX-License-Identifier: GPL-3.0-or-later
```

Comment:
I would use HTML style comments, instead. e.g.

``` markdown
<!--
GNU General Public License v3.0+ (see LICENSES/GPL-3.0-or-later.txt or https://www.gnu.org/licenses/gpl-3.0.txt)
SPDX-License-Identifier: GPL-3.0-or-later
-->
```

Take a look at https://github.com/ansible-collections/community.docker/blob/b60d7d61b0a5583db185b3b5a4487846fcd94c56/.azure-pipelines/README.md to see how this currently looks

Reply:
I'll look at this more later. That was just the first thing that I noticed.

Reply:
I was using https://stackoverflow.com/a/20885980 as a reference. Maybe it needs another newline between the two comments, I'll try that. (one minute later) Hmm, that doesn't work either (https://github.com/ansible-collections/community.docker/blob/f1fed46fb528934f6ee510354d937571b2d45257/README.md).

Ok, a bit more experimenting showed that the brakets around the comment text are important. But once I add these, reuse cannot parse `[comment]: # (SPDX-License-Identifier: GPL-3.0-or-later)` anymore: `reuse._util - ERROR - Could not parse 'GPL-3.0-or-later)'`. What a mess. I've switched to HTML comments now, I simply hope that this will also work with Galaxy's README Markdown renderer...

Reply:
>  I've switched to HTML comments now, I simply hope that this will also work with Galaxy's README Markdown renderer...

[Have your worries assuaged](https://galaxy.ansible.com/gotmax23/aur_test) :)

Reply:
Thanks for testing this!